DEV-INTRO.md

```diff
@@ -1,59 +1,187 @@
 
-#### From GitHub
+* setup .env file
+* create folders to hold the data
+* load the postgresql database
+* load some test data
+* copy all R packages (optional but recommended) 
+* setup for web folder development (optional)
 
+#### .env file
+
+You can copy the [`env.example`](docker/env.example) file as .env in your pecan folder. The variables we want to modify are:
+
+* `COMPOSE_PROJECT_NAME` set this to pecan, the prefix for all containers
+* `PECAN_VERSION` set this to develop, the docker image we start with
+
+At the end you should see the following if you run the following command `egrep -v '^(#|$)' .env`
+
+```
+COMPOSE_PROJECT_NAME=pecan
+PECAN_VERSION=develop
+```
+
+#### folders
+
+Next we will create the folders that will hold all the data for the docker containers using: `mkdir -p volumes/{lib,pecan,portainer,postgres,rabbitmq,traefik}`. The `volumes` folder will be ignored by git. You can create these at any location, however you will need to update the `docker-compose.dev.yml` file. The subfolders are used for the following:
+
+- **lib** holds all the R packages for the specific version of PEcAn and R. This folder will be shared amongst all other containers, and will contain the compiled PEcAn code.
+- **pecan** this holds all the data, such as workflows and any downloaded data.
+- **portainer** if you enabled the portainer service this folder is used to hold persistent data for this service
+- **postgres** holds the actual database data. If you want to backup the database, you can stop the postgres container, zip up the folder.
+- **rabbitmq** holds persistent information of the message broker (rabbitmq). 
+- **traefik** holds persisent data for the web proxy, that directs incoming traffic to the correct container.
+
+These folders will hold all the persistent data for each of the respective containers and can grow. For example the postgres database is multiple GB. The pecan folder will hold all data produced by the workflows, including any downloaded data, and can grow to many giga bytes.
+
+#### postgresql database
+
+First we bring up postgresql (we will start RabbitMQ as well since it takes some time to start): `docker-compose -f docker-compose.yml -f docker-compose.dev.yml up -d postgres rabbitmq`. This will start postgresql and rabbitmq. We need to wait for a few minutes (you can look at the logs using `docker-compose logs postgres`) to see if it is ready.
+
+Once the database has finished starting up we will initialize the database. Before we run the container we want to make sure we have the latest database infomration, you can do this with `docker pull pecan/db`, which will make sure you have the latest version of the database ready. Now you can load the database using: `docker run --rm --network pecan_pecan pecan/db` (in this case we use the latest image instead of develop since it refers to the actual database data, and not the actual code). Once that is done we create two users for BETY:
```

Comment:
```suggestion
Once the database has finished starting up we will initialize the database. Before we run the container we want to make sure we have the latest database information, you can do this with `docker pull pecan/db`, which will make sure you have the latest version of the database ready. Now you can load the database using: `docker run --rm --network pecan_pecan pecan/db` (in this case we use the latest image instead of develop since it refers to the actual database data, and not the actual code). Once that is done we create two users for BETY:
```

Reply:
this is just typo in "infomration" corrected